src/core/components/ToolCallProgressCards.tsx

```diff
@@ -12,12 +13,11 @@ export const ToolCallProgressCards = ({
     toolCallStreamStore.subscribe,
     toolCallStreamStore.getSnapShot,
   );
-  const { registry } = useAppContext();
 
   if (!toolCallStreams.length) return null;
 
   return toolCallStreams.map((toolCall) => {
-    const operation = registry.get(toolCall.function.name ?? '');
+    const operation = messageRegistry.get(toolCall.function.name ?? '');
```

Comment:
@downing034 is it this component that is not showing?

Reply:
@samcode206 - it's the in progress tx display that doesn't show when send

Reply:
i am not seeing this class having the inProgressComponent set up anymore when i log it 
<img width="1618" alt="Screen Shot 2025-02-04 at 11 06 28 AM" src="https://github.com/user-attachments/assets/3af85264-64b4-496f-8b6b-ed1b28d34aa0" />